interface/src/ui/DialogsManager.cpp

```diff
@@ -110,10 +110,29 @@ void DialogsManager::setDomainConnectionFailureVisibility(bool visible) {
     }
 }
 
+void DialogsManager::requestMetaverseLoginState() {
+    DialogsManager::setMetaverseLoginState();
+}
+
+void DialogsManager::requestDomainLoginState() {
+    DialogsManager::setDomainLoginState();
+}
```

Comment:
Why these synonyms for the "set" methods? (Is there a "TODO" here that should be marked?)

Reply:
There might have been, but I cannot remember now. We'll find out when we need it again. :)